docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 sphinx==8.1.3
 sphinx_rtd_theme==3.0.1
-sphinx-copybutton>=0.5.0
\ No newline at end of file
+sphinx-copybutton>=0.5.0
+aiohttp
```

Comment:
@pcriadoperez do we need aiohttp here?

Reply:
I was surprised too. But without it would fail to import the modules to generate the docs.

Reply:
weird 😅